src/docs/product/accounts/quotas/manage-event-stream-guide.mdx

```diff
@@ -1,175 +1,188 @@
+While SDK configuration requires changes to your source code and depends on your next deployment, server-side filters can be easily configured per project in the "Data Filters" section of **[Project] > Settings > Inbound Filters**. Learn more in [Inbound Filters](/product/accounts/quotas/#inbound-data-filters).
 
-Once applied, you can track the filtered events (numbers and cause) using the graph provided at the top of the Inbound Data Filters view.
+Once applied, you can track the filtered events (numbers and cause) using the graph provided at the top of the "Inbound Filters" page.
 
 ![Built-in Inbound Filters](manage-event-stream-03.png)
 
 ## 4. Event Grouping
 
-Proper event grouping is essential to maintain a meaningful Issue Stream and reduce redundant notifications. Sentry groups similar _Events_ into unique _Issues_ based on their _Fingerprint_. An event's fingerprint relies firstly on its **stack trace**.
+Proper event grouping is essential to maintaining a meaningful set of issues and to reduce redundant notifications. Sentry groups similar _events_ into unique _[issues](/product/issues/)_ based on their _fingerprint_. An event's fingerprint relies firstly on its _stack trace_.
 
-With **JavaScript** errors, a minified source code might result in a nondeterministic stack trace that could mess up associated event grouping. Make sure Sentry has access to your `Source Maps` and minified artifacts. For more information see [Uploading Source Maps](/platforms/javascript/sourcemaps/).
+With JavaScript errors, a minified source code might result in a nondeterministic stack trace that could negatively affect associated event grouping. Make sure Sentry has access to your source maps and minified artifacts. Learn more in [Source Maps](/platforms/javascript/sourcemaps/).
 
 ![JavaScript stack trace without source maps](manage-event-stream-04.png)
+
 ![JavaScript stack trace with source maps](manage-event-stream-05.png)
 
-For more information on the Fingerprint algorithm and customizing event grouping see [Grouping & Fingerprints](/product/data-management-settings/event-grouping/).
+For more information on the fingerprint algorithm and customizing event grouping, check out our [Issue Grouping docs](/product/data-management-settings/event-grouping/).
 
-## 5. Applying Workflows
+## 5. Resolve or Delete Issues {#5-applying-workflows}
 
-Now that your event stream is fine-tuned to reflect real problems in your code, it's an excellent practice to react to errors as they happen. If an issue reflects a real problem in your code, resolve it; otherwise--- discard.
+Now that your error events have been fine-tuned to reflect real problems in your code, it's an excellent practice to react to errors as they happen. If an issue reflects a real problem in your code, resolve it; otherwise, discard it.
```

Comment:
```suggestion
Now that your error events have been fine-tuned to reflect real problems in your code, it's an excellent practice to react to errors as they happen. If an issue reflects a real problem in your code, resolve it; otherwise, delete and discard it.
```